convert.go

```diff
@@ -151,3 +151,34 @@ func ToString(arg any, timeFormat ...string) string {
 		return fmt.Sprint(arg)
 	}
 }
+
+// AppendInt appends the string representation of the int n to dst and returns the extended buffer.
+func AppendInt(dst []byte, n int) []byte {
```

Comment:
I think we need a separate function for `AppendUint` that uses `uint` as param, else we won't be able to handle values bigger than `2,147,483,647`

Reply:
The `math` module in Golang defines these:

```go
MaxUint32 = 1<<32 - 1           // 4294967295
MaxUint64 = 1<<64 - 1           // 18446744073709551615
```